test/razorpay/test_subscription_registration.rb

```diff
@@ -0,0 +1,59 @@
+require 'test_helper'
+
+module Razorpay
+  # Tests for Razorpay::Addon
+  class RazorpayAddonTest < Minitest::Test
+    class SubscriptionRegistration < Razorpay::Entity; end
```

Comment:
why is the `RazorpayAddonTest ` class used here?

Reply:
fixed class name